### PR TITLE
eleventy-cli

### PR DESCRIPTION
**Pull request by yavko**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adds Eleventy Cli Support
**What is the current behavior? (You can also link to an open issue here)**
No support
**What is the new behavior (if this is a feature change)?**
support
**Additional info:**
4th contribution

### CODE REVIEW

File: dev/eleventy.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      dependsOn: "--serve",
+      args: {
+        name: "port",
+      },
+    },
+    {
+      description: "Whitelist only certain template types.",
+      name: "--formats",
+      args: {
+        name: "Template Names Seperated by ,",
```
Comment (Monkeyank) [inline dev/eleventy.ts:61]: I think we can provide a list of suggestions, such as `md,html,ejs`

File: dev/eleventy.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    {
+      description: "Run web server on port (default 8080) and watch them too.",
+      name: "--serve",
+    },
+    {
+      description: "Write HTML output to this folder.",
+      name: "--port",
+      dependsOn: "--serve",
+      args: {
+        name: "port",
```
Comment (Monkeyank) [inline dev/eleventy.ts:54]: Maybe add a `default: "8080"`?

**Review by Monkeyank — CHANGES_REQUESTED** [review]

File: dev/eleventy.ts (hunk 1 above)
Comment (yavko) [reply dev/eleventy.ts:61]: ok

File: dev/eleventy.ts (hunk 2 above)
Comment (yavko) [reply dev/eleventy.ts:54]: ok

File: dev/eleventy.ts (hunk 1 above)
Comment (yavko) [reply dev/eleventy.ts:61]: Idk how tho, I read the docs and i still don't get how

Comment (Monkeyank) [reply dev/eleventy.ts:61]: You can add a prop underneath args. Here's an example:

```
args: {
     name: "Template Names Separated by,",
     suggestions: ["md", "html", "eje"]
}
```

Comment (yavko) [reply dev/eleventy.ts:61]: ok thanks

Comment (fedeci) [reply dev/eleventy.ts:61]: However that does not allow for comma separated values, right?

Comment (yavko) [reply dev/eleventy.ts:61]: idk

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for this PR!

**Comment by yavko** [comment]
YAYAYA PR 4

**Pull request merged**